lib/guardian/topic_guardian.rb

```diff
@@ -48,8 +48,11 @@ def can_publish_topic?(topic, category)
   def can_create_topic?(parent)
     is_staff? ||
       (
-        user && user.trust_level >= SiteSetting.min_trust_to_create_topic.to_i &&
-          can_create_post?(parent) && Category.topic_create_allowed(self).limit(1).count == 1
+        user &&
+          (
+            user.trust_level >= SiteSetting.min_trust_to_create_topic.to_i ||
+              user.in_any_groups?(SiteSetting.create_topic_allowed_groups_map)
```

Comment:
I'm not sure this will always work. Specifically when a value is stored in the DB we'll get a string like `"14"` or (with multiple groups) `"14|1"`. Only with the default do we get a number I believe.

Reply:
Never mind, didn't see the `_map`. 😅